### PR TITLE
[geo-index][covering] Fixed cells covering. Use more predictive criteria.

### PR DESCRIPTION
**Pull request by vng**
Old bug. Better cells fitting into geometry. May increase index file size a little bit.

### CODE REVIEW

File: geometry/covering_utils.hpp [hunk 1]
```diff
@@ -101,16 +101,21 @@ void CoverObject(IntersectF const & intersect, uint64_t cellPenaltyArea, CellIdC
 
   uint64_t subdivArea = 0;
   for (size_t i = 0; i < subdiv.size(); ++i)
-    subdivArea += my::sq(uint64_t(1 << (CellIdT::DEPTH_LEVELS - 1 - subdiv[i].Level())));
+    subdivArea += my::sq(uint64_t(1 << (cellDepth - 1 - subdiv[i].Level())));
 
   ASSERT(!subdiv.empty(), (cellPenaltyArea, out, cell));
 
-  if (subdiv.empty() || cellPenaltyArea * (int(subdiv.size()) - 1) >= cellArea - subdivArea)
+  //if (cellPenaltyArea * (int(subdiv.size()) - 1) >= cellArea - subdivArea)
```
Comment (ygorshenin) [inline geometry/covering_utils.hpp]: Please, remove this if.

**Comment by ygorshenin** [comment]
LGTM, but someone else must approve.

**Comment by ygorshenin** [comment]
DUDE JTTP

**Comment by vng** [comment]
PTAL. What's wrong with formatting here?

**Comment by mpimenov** [comment]
LGTM

**Comment by mpimenov** [comment]
> What's wrong with formatting here
Nothing. The bot does not complain.

**Pull request merged**